### PR TITLE
add ORAS proposals to LFX mentorship term2

### PR DESCRIPTION
**Pull request by FeynmanZhou**
@nate-double-u I want to add two ORAS proposals to the LFX mentorship term 2. Thanks

### CODE REVIEW

**Comment by nate-double-u** [comment]
Thanks for this @FeynmanZhou! I thought you were a maintainer on ORAS -- but i don't see you on https://github.com/cncf/foundation/blob/main/project-maintainers.csv#L956 now.

@TerryHowe, as the maintainer listed, could you give this PR a thumbs up? You don't need to be a mentor on each project, but we do need a maintainer's approval for all the projects.

**Review by TerryHowe — APPROVED** [review]

**Review by aliok — APPROVED** [review]
Thanks @FeynmanZhou @qweeah @TerryHowe @asmitbm !

**Pull request merged**
